bootstrap.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set +x
+
+# Ask the user for the installation directory
+read -p "Enter the installation directory (e.g., '.' or 'docs'): " install_directory
```

Comment:
Just a suggestion: on the basis of "favour convention over configuration", could we make the use of `docs` stronger than a mere option? The default, say, or "`docs` is strongly recommended"?

Reply:
If you have a doc-only repo, you wouldn't want a `docs` folder in there.
Other repos might already have a `doc` folder, and to avoid breaking any links, it's better to keep it then.

Reply:
Agreed, that's why I specified "for documentation alongside code". Either way, readme.rst L23 needs to be consistent with it.

If they have an existing ``doc`` folder are we recommending that they add SP in a separate one (say, ``docs``), or add it to the existing one and do whatever's needed to consolidate them?

Reply:
> If they have an existing `doc` folder are we recommending that they add SP in a separate one (say, `docs`), or add it to the existing one and do whatever's needed to consolidate them?

The latter - you want to reuse the existing files (someone might be linking to the files on GitHub).

Reply:
OK I've updated L23

Reply:
> > If they have an existing `doc` folder are we recommending that they add SP in a separate one (say, `docs`), or add it to the existing one and do whatever's needed to consolidate them?
> 
> The latter - you want to reuse the existing files (someone might be linking to the files on GitHub).

My preference too - thanks for clarifying, [Ruth](@ru-fu ).